src/js/base/module/Editor.js

```diff
@@ -47,8 +47,12 @@ define([
         }
         context.triggerEvent('keydown', event);
 
-        if (options.shortcuts && !event.isDefaultPrevented()) {
-          self.handleKeyMap(event);
+        if (!event.isDefaultPrevented()) {
+          if (options.shortcuts) {
+            self.handleKeyMap(event);
+          } else {
+            return self.preventDefaultEditableShortCuts(event);
```

Comment:
@lqez 
I thought we don't have to return boolean value here. Is there reason for this code?